components/src/core/components/InputField/_variables.scss

```diff
@@ -9,6 +9,7 @@ $oxd-input-group-message-font-weight: 600 !default;
 $oxd-input-group-label-margin-bottom: 0.4rem !default;
 $oxd-input-group-label-icon-margin-right: 4px !default;
 
+$oxd-input-field-hint-margin: 0.25rem 0 0 0;
```

Comment:
Can we use px instead of rem here. Then if the base font size is changes, the gap between the 2 elements will be the same.

Reply:
Changed margin to px, recommended by Mike as well.